operators/hpe-csi-operator/deploy/olm-catalog/hpe-csi-driver-operator/2.0.0/hpe-csi-driver-operator.v2.0.0.clusterserviceversion.yaml

```diff
@@ -21,7 +21,6 @@ metadata:
               "backend": "192.168.1.1",
               "create": true,
               "password": "admin",
-              "serviceName": "nimble-csp-svc",
```

Comment:
Please leave this in as "backend-csp-svc"

Reply:
This is handled internally in the templates depending on the backendtype ,so no part of values.yaml so removed.